lib/Chat/Changelog/Manager.php

```diff
@@ -119,6 +119,10 @@ public function getChangelogs(): array {
 			$this->l->t('- Join the same conversation and call from multiple devices'),
 			$this->l->t('- Send voice messages, share your location or contact details'),
 			$this->l->t('- Add groups to a conversation and new group members will automatically be added as participants'),
+			$this->l->t('New in Talk %s', ['13']),
+			$this->l->t('- You can now select and disable your devices before joining a call'),
```

Comment:
I would rather say `A preview of your audio and video is shown before joining a call`.